french/maps/conv/sienna_khan.rpy

```diff
@@ -10,179 +10,179 @@ translate french sienna_redcliffs_prison_01_a58f706d:
 translate french sienna_redcliffs_prison_01_3845cef7:
 
     # d torch neutral "Hmm...{w=0.5} Well, in short, while we couldn't get a precise scan we managed to find a system of natural caves in the rock of the Red Cliffs."
-    d torch neutral ""
+    d torch neutral "Hmm...{w=0.5} Bon, de manière courte, même si nous n'avons pas pu avoir un scan précis, nous sommes parvenus à trouver un système de cavernes naturelles dans la roche des Red Cliffs."
```

Comment:
in short -> en résumé
nous trop formel (vu que dans l'intro ils ont give up les formalités)